bangazonapi/views/profile.py

```diff
@@ -177,15 +177,15 @@ def cart(self, request):
             try:
                 open_order = Order.objects.get(
                     customer=current_user, payment_type=None)
-                line_items = OrderProduct.objects.filter(order=open_order)
-                line_items = LineItemSerializer(
-                    line_items, many=True, context={'request': request})
+                lineitems = OrderProduct.objects.filter(order=open_order)
+                lineitems = LineItemSerializer(
```

Comment:
Take another look at this one. 

Right now all the variables are the same, so instead of getting duplicates, the `lineitems` variable is simply being overwritten multiple times. This is a problem, because we've got code in multiple places, trying to do the same exact job. 

Where should you look? 
* Find the code responsible for serializing the url, created_date, customer, paymenttype, etc. being serialized. HINT: In one of the serializers.
* What else is that code serializing? HINT: lineitems
* Do we need to serialize lineitems in more than one place? HINT: Nope!

Review the code and remove the unnecessary code. HINT: It's much easier to figure out which code is needed and which code is duplicating work if you change the spelling back to the way it was.